exp/views/responses.py

```diff
@@ -1854,32 +1801,42 @@ def get(self, request, *args, **kwargs):
         return response
 
 
-class StudyDemographicsDownloadDictCSV(StudyDemographics):
+class StudyDemographicsDictCSV(DemographicDownloadMixin, generic.list.ListView):
     """
     Hitting this URL downloads a data dictionary for participant demographics in in CSV format.
     Does not depend on any actual data.
     """
 
-    def get(self, request, *args, **kwargs):
-        study = self.get_object()
-        header_options = self.request.GET.getlist("demooptions")
-        cleaned_data = self.build_all_demographic_dict_csv(header_options)
+    def render_to_response(self, context):
+        header_options = self.request.GET.getlist("demo_options")
+        these_headers = get_demographic_headers(header_options)
```

Comment:
`these_headers` isn't really a great variable name - why does `demographic_headers` not work?

Reply:
changed to `headers_for_download` (idea is that these are the particular headers selected based on the request & the available headers, so `demographic_headers` is more ambiguous)